tests/signing/tests.py

```diff
@@ -178,6 +178,39 @@ def test_invalid_sep(self):
             with self.assertRaisesMessage(ValueError, msg % sep):
                 signing.Signer(sep=sep)
 
+    def test_verify_with_non_default_key(self):
+        old_signer = signing.Signer('secret')
+        new_signer = signing.Signer('newsecret', fallback_keys=['secret'])
+        signed = old_signer.sign('abc')
+        self.assertEqual(new_signer.unsign(signed), 'abc')
+
+    def test_sign_unsign_multiple_keys(self):
+        """Make sure the default key is a valid verification key."""
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Make sure" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).
```suggestion
        """The default key is a valid verification key."""
```